awx/main/utils/common.py

```diff
@@ -699,12 +699,25 @@ def parse_yaml_or_json(vars_str, silent_failure=True):
     return vars_dict
 
 
-def get_cpu_capacity(raw=None):
+def get_cpu_effective_capacity(cpu_count):
```

Comment:
These env vars are still so confusing to me, but I'm glad to see some of this refactored out a helper method

Reply:
yeah, it's still not great. I expect these will have continued revisions as we keep going.